cernopendata/modules/fixtures/data/docs/delphi-about/delphi-about.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
I would also add the logo to the footer, and the link to the delphi-about on the top right corner. I'll send some instructions on how to do that.